### PR TITLE
Turn imports test back on - sans tools

### LINKED ISSUE #4299: Migration: Turn tests back on after NuGet imports goes in

### PR DESCRIPTION
**Pull request by brthor**
Fixes #4299

@livarcocc @piotrpMSFT @jgoshi @eerhardt

### CODE REVIEW

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs [hunk 1]
```diff
@@ -86,11 +80,28 @@ public void It_migrates_dotnet_new_console_with_identical_outputs()
...
+            var globalDirectory = Path.Combine(AppContext.BaseDirectory, "newwebtest");
+            var projectDirectory = Path.Combine(globalDirectory, "project");
+            if (Directory.Exists(projectDirectory))
+            {
+                Directory.Delete(projectDirectory, true);
+            }
+            Directory.CreateDirectory(projectDirectory);
+
+            WriteGlobalJson(globalDirectory);
+            DotnetNew(projectDirectory, "web");
```
Comment (livarcocc) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: This won't work for long. Should we have a web test asset to be used here? This is just adding debt for when we switch the templates.

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs [hunk 2]
```diff
@@ -86,11 +80,28 @@ public void It_migrates_dotnet_new_console_with_identical_outputs()
             VerifyAllMSBuildOutputsRunnable(projectDirectory);
         }
 
-        [Fact(Skip="https://github.com/dotnet/cli/issues/4299")]
+        [Fact]
         public void It_migrates_dotnet_new_web_with_outputs_containing_project_json_outputs()
```
Comment (livarcocc) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Should we say it migrates web projects without tools?

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs [hunk 3]
```diff
@@ -86,11 +80,28 @@ public void It_migrates_dotnet_new_console_with_identical_outputs()
...
+            WriteGlobalJson(globalDirectory);
+            DotnetNew(projectDirectory, "web");
+            var projectJsonFile = Path.Combine(projectDirectory, "project.json");
+
+            var projectJson = JObject.Parse(File.ReadAllText(projectJsonFile));
+            projectJson.Remove("tools");
+
+            File.WriteAllText(projectJsonFile, projectJson.ToString());
+
+            Directory.CreateDirectory(projectDirectory);
```
Comment (eerhardt) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Isn't this directory already created?

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs [hunk 4]
```diff
@@ -101,6 +112,16 @@ public void It_migrates_dotnet_new_web_with_outputs_containing_project_json_outp
...
+            var file = Path.Combine(globalDirectory, "global.json");
+
+            File.WriteAllText(file, @"
+            {
+                ""projects"": [ ]
+            }");
+        }
+
         [Theory]
         // TODO: Enable this when X-Targeting is in
```
Comment (eerhardt) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Do we have a bug for this TODO?  cross targeting is in.

**Comment by eerhardt** [comment]
:shipit:

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs (hunk 4 above)
Comment (brthor) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: No I saw this too, turned it back on but it's failing, going to fix it today and make a separate PR. There's another test in this file that needs to be turned on too

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs (hunk 2 above)
Comment (brthor) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: sure

**Comment by brthor** [comment]
@dotnet-bot test this please

**Pull request closed without merging**